lib/main.coffee

```diff
@@ -10,14 +10,15 @@ module.exports =
       default: 'foodcritic'
     extraArgs:
       type: 'string'
+      default: ''
 
   activate: ->
     @subscriptions = new CompositeDisposable
     @MessageRegexp = null
-    @subscriptions.add atom.config.observe 'foodcritic.executablePath', (executablePath) =>
-      @executablePath = executablePath
-    @subscriptions.add atom.config.observe 'foodcritic.extraArgs', (extraArgs) =>
-      @extraArgs = extraArgs
+    @subscriptions.add atom.config.observe 'foodcritic.executablePath', =>
```

Comment:
`'foodcritic.` this should be `'linter-foodcritic.`, aka the package name in manifest